spark/src/main/scala/io/delta/sql/DeltaSparkSessionExtension.scala

```diff
@@ -80,7 +80,11 @@ import org.apache.spark.sql.internal.SQLConf
  *
  * @since 0.4.0
  */
-class DeltaSparkSessionExtension extends (SparkSessionExtensions => Unit) {
+// V1 entry point class for backwards compatibility with Spark V1 APIs.
+class V1DeltaSparkSessionExtension extends AbstractDeltaSparkSessionExtension
```

Comment:
why V1ClassName, and not ClassNameV1? 
we have been following the pattern of version at the end... isnt it? DeltaTableV2?